arch/arm/soc/nxp_kinetis/kw2xd/Kconfig.series

```diff
@@ -0,0 +1,16 @@
+# Kconfig - Kinetis KW2xD MCU series
+#
+# Copyright (c) 2017, Phytec Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_KINETIS_KW2XD
```

Comment:
My original intent when I added KW41Z was that KW2xD would be part of the same series (e.g., the Kinetis W-series). That's why I didn't select a specific Cortex M core (M0+ vs. M4) with SOC_SERIES_KINETIS_KWX. Your thoughts on merging kw2xd into kwx series?